snyk/code-client
# fix: case-sensitive file extensions filtering [ZEN-371] [ZEN-614]

- [x] Linked to Jira ticket

#### What does this PR do?
Ignores file extension casing when filtering out for supported files before uploading for analysis.
Current tests are failing due to https://github.com/snyk/sast-analysis-api/pull/328

#### Where should the reviewer start?

#### How should this be manually tested?
Build the lib with CLI, then run a CLI scan on any folder with a single file having an uppercase file extension. This file should not be filtered out for analysis, providing Code results back.

#### Any background context you want to provide?
We're already case insensitive for file extensions around analysis in IDE. This PR addresses CLI. Next is to be case insensitive within Git and PR checks flow.

#### Screenshots

#### Additional questions
